src/progress_bar_exercise/ProgressBarExercise.js

```diff
@@ -18,5 +20,47 @@ export default ProgressBarExercise;
 // ----------------------------------------------------------------------------------
 
 const Solution = () => {
-  return <div>Add solution here</div>;
+  const [loading, setLoading] = useState()
+  return (
+    <div {...useCss(css)}>
+      <ProgressBar loading={loading} />
+      <div className='buttonWrapper'>
+        <button onClick={() => setLoading(true)}>
+          {loading ? "LOADING..." : "START REQUEST"}
+        </button>
+      </div>
+      <div className='buttonWrapper'>
+        <button onClick={() => setLoading(false)} className='red'>
+          FINISH REQUEST
+        </button>
+      </div>
+    </div>
+  )
 };
+
+const css = `
+  & button {
+    font-weight: 500;
+    border: var(--green) 1px solid;
+    color: var(--green);
+    padding: .8rem 1.2rem;
+    border-radius: 2rem;
+  }
+  & button:hover {
+    border-width: 2px;
+    margin: -1px;
```

Comment:
negative margins trick helps avoid weird space management issues inherent to changing border